src/components/BibleDropDown/BookContext.js

```diff
@@ -7,16 +7,14 @@ const BookContextProvider = (props) => {
   const [books, setBooks] = useState([]);
 
   useEffect(() => {
-    API.get('https://www.api.vachanonline.net/v1/booknames')
+    API.get('lookup/bible/books?limit=100')
       .then((response) => {
-        setBooks(
-          response.data[0].bookNames.sort((a, b) => a.book_id - b.book_id)
-        );
+        setBooks(response.data);
       })
       .catch((err) => console.log(err));
   }, []);
   return (
-    <BookContext.Provider value={{ books }}>
+    <BookContext.Provider value={{ books: books }}>
```

Comment:
we can undo this to value={{ books }} 
That is a short form if both key and value are same